docs/tutorials/02_speed.ipynb

```diff
@@ -0,0 +1,1074 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    em.setup(X, y)
</pre><p>earlier the setup info was relevant but at this step the changed parameter (number of models) is not part of the output. For more clarity in the tutorial should we add a 'verbose' option to the setup method that we can put to 'false' for this purpose?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/autoemulate/pull/231/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
yes, agreed. I added an argument `print_setup()`, as `verbose` does control a lot of other logging stuff.